paket.lock

```diff
@@ -76,11 +76,19 @@ NUGET
     toastr (2.1.1)
       jQuery (>= 1.6.3)
     UnionArgParser (0.8.6)
-    xunit (1.9.2)
-    xunit.extensions (1.9.2)
-      xunit (1.9.2)
-    xunit.runners (1.9.2)
+    xunit (2.0.0)
```

Comment:
we can't do that right now. It still doesn't work on mono.

I assume you just want to update YaafFSharpScripting, right?